### ISSUE TITLE
Add reminder when a PR is ready to be merged

### ISSUE THREAD

**Issue by marchersimon** [tldr-pages/tldr-bot#13]
When you approve a PR which can't yet be merged, because it was just opened a few hours ago, you prabably mark the notification as done. This either leads to forgetting about the PR and leaving it unmerged for a long time or merging it too early to prevent this problem. This could be solved by a notification from tldr-bot with these requirements:

* The PR has enough approvals and everyone, who requested changes has approved.
* The PR is at least 2 days old and has had no activity for the last 12 hours.
* If there are enough approvals, but not everyone who requested changes approved, set the minimum time to 4 days.

**Comment by sbrl** [comment]
Sounds good! We could absolutely implement this. It could maybe scan all open PRs every 24 hours?

**Comment by marchersimon** [comment]
Sure. I'd love to do this myself, but if it's more advanced than `Hello World` I'll probably fail miserably 😆.

Does everyone agree with the ruleset for this new action?

**Comment by bl-ue** [comment]
>If there are enough approvals, but not everyone who requested changes approved, set the minimum time to 4 days.

Maybe that should be extended to 7 days, because we're not always that active...

**Comment by waldyrious** [comment]
Isn't this covered by the stale flow?

**Comment by marchersimon** [comment]
We only get notifications from tldr-bot after a month, which is a pretty long time to leave a finished PR unmerged.

**Comment by sbrl** [comment]
Yeah, I'd suggest if a PR has enough approvals and and is idle for at least 2 days then tldr-bot posts a single reminder. If someone has requested changes and has yet to approve, wait 7 days before reminding.

Currently GitHub actions should notify on 14 days, and then auto-close at 30 days.

**Comment by marchersimon** [comment]
Maybe we could even add a custom reminder for specific cases, like 
> @tldr-bot remind in 3 days
